covid_xprize/examples/prescriptors/neat/unused_geos.txt

```diff
@@ -1,43 +0,0 @@
-Azerbaijan__nan
-Benin__nan
```

Comment:
This file no longer applies, since the predictors were updated to handle these regions long ago.